Can't resolve quadTree.js, interaction.js when using ES6 to import highcharts

#### Expected behaviour
Expect these files to be resolved correctly

#### Actual behaviour
ERROR in ./node_modules/highcharts/es-modules/modules/networkgraph/layouts.js
Module not found: Error: Can't resolve 'QuadTree.js' in '/Users/chhay/Desktop/Plotter/node_modules/highcharts/es-modules/modules/networkgraph'
ERROR in ./node_modules/highcharts/es-modules/modules/networkgraph/layouts.js
Module not found: Error: Can't resolve 'integrations.js' in '/Users/chhay/Desktop/Plotter/node_modules/highcharts/es-modules/modules/networkgraph'

#### Live demo with steps to reproduce
<!-- template: https://jsfiddle.net/highcharts/LLExL/ -->
- Use ES6 to import highchart to use

#### Product version
<!--- Highcharts, Highstock or Highmaps plus version number -->
Highcharts V 7.1.3

#### Affected browser(s)
Using it on Electron

--- a/js/modules/networkgraph/layouts.js
+++ b/js/modules/networkgraph/layouts.js
@@ -11,8 +11,8 @@
 import H from '../../parts/Globals.js';
 import U from '../../parts/Utilities.js';
 var defined = U.defined;
-import 'integrations.js';
-import 'QuadTree.js';
+import './integrations.js';
+import './QuadTree.js';
 var pick = H.pick, addEvent = H.addEvent, Chart = H.Chart;
 /* eslint-disable no-invalid-this, valid-jsdoc */
 H.layouts = {

--- a/ts/modules/networkgraph/layouts.ts
+++ b/ts/modules/networkgraph/layouts.ts
@@ -143,8 +143,8 @@ import U from '../../parts/Utilities.js';
 var defined = U.defined;
 
 
-import 'integrations.js';
-import 'QuadTree.js';
+import './integrations.js';
+import './QuadTree.js';
 
 var pick = H.pick,
     addEvent = H.addEvent,

